requirements-docs.txt

```diff
@@ -623,25 +619,23 @@ xmltodict==0.12.0
     #   -r requirements-docs.in
     #   -r requirements-tests.txt
     #   moto
-zipp==3.7.0
-    # via importlib-metadata
-zope.component==5.0.1
+zope-component==5.0.1
```

Comment:
I found in #3969 that leaving these alone (not switching `-` to `.`) makes the build pass. I'm unclear what the root cause of this behavior is, though.